### ISSUE TITLE
Error loading CSS.sublime-syntax: Unable to find CSS.sublime-syntax in Packages/PHP/PHP.sublime-syntax

### ISSUE THREAD

**Issue by chavab1** [erichard/SublimePHPCompanion#99]
I get the following error after I installed this plugin:

```
Error loading CSS.sublime-syntax: Unable to find CSS.sublime-syntax in Packages/PHP/PHP.sublime-syntax
Error loading scope:source.css: Unable to find syntax file for scope source.css in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 344 column 28 in Packages/PHP/PHP.sublime-syntax
no such context CSS.sublime-syntax#rule-list-body at line 363 column 28 in Packages/PHP/PHP.sublime-syntax
```

**Comment by valentinoper** [comment]
I dont know what plugin you have installed, but recently I opened my codeigniter project and noticed that sublime wasnt showing me function definitions on hover. So i opened up the console only to find this error Error loading CSS.sublime-syntax: Unable to find CSS.sublime-syntax in Packages/PHP/PHP.sublime-syntax

Anybody has a fix?

**Comment by LeaveAirykson** [comment]
I had the same error recently and sublime sometimes just crashed while working in css files. I found out that the CSS Package was disabled. Enabling it and restarting sublime resolved the issue.

**Comment by chavab1** [comment]
You're absolutely correct @LeaveAirykson. I didn't have the CSS Package enabled either, and enabling it fixed it. The funny thing is that I tried to install and enable CSS3 package and I still got the same error. It was only after disabling it and enabling regular CSS that the error disappeared. 

However, unlike you, I only saw this error after I installed/enabled PHP Companion.

So I'm not sure what package is causing the problem.

Either way, thanx for the help.

BTW: I'm not sure if I should close this since the error still remains with the latest CSS package.
